src/ui/components/SidePanel.tsx

```diff
@@ -96,28 +97,30 @@ export default function SidePanel() {
   const selectedPrimaryPanel = useInitialPrimaryPanel();
   const [replayInfoCollapsed, setReplayInfoCollapsed] = useState(false);
   const [eventsCollapsed, setEventsCollapsed] = useState(false);
+  const events = useAppSelector(getFlatEvents);
 
   const items: any[] = [];
 
   // if (recording?.metadata?.test?.tests?.length) {
-  items.push(
-    {
-      header: "Info",
-      buttons: resolveRecording ? <StatusDropdown /> : null,
-      className: "replay-info",
-      component: <ReplayInfo />,
-      opened: !replayInfoCollapsed,
-      onToggle: () => setReplayInfoCollapsed(!replayInfoCollapsed),
-    },
-    {
+  items.push({
+    header: "Info",
+    buttons: resolveRecording ? <StatusDropdown /> : null,
+    className: "replay-info",
+    component: <ReplayInfo />,
+    opened: !replayInfoCollapsed,
+    onToggle: () => setReplayInfoCollapsed(!replayInfoCollapsed),
+  });
+
+  if (events && Array.isArray(events) && events.length > 0) {
```

Comment:
I _think_ `getFlatEvents` will _always_ return `ReplayEvent[]` – is there any reason we're checking for undefined and is-array here?

Reply:
Short answer: I was flailing and I will revert this.

Long answer: when you called out the Typescript error I was trying to figure out how it snuck in. I think it was because Ryan also committed code at the same time and I didn't see that the merge changed stuff. So this was me troubleshooting, and I thought it'd be fine to keep in. But happy to pull it back out.